_data/sponsors.yml

```diff
@@ -4,11 +4,12 @@
 # 2017: ["takealot", "google", "oracle", "offerzen", "allan_gray", "fastly", "jumo", "amazon", "impact_radius"]
 # 2018" ["takealot", "amazon", "allan_gray", "offerzen", "impact_radius", "oracle", "google", "luno", "smyte"]
 # 2019" ["impact", "estalea", "elastic", "offerzen", "microsoft", "takealot", "dimensiondata", "luno", "amazon", "allan_gray", "equal_experts", "aruba"]
+# 2020" ["luno"]
```

Comment:
this is just a comment, but it's wrong because elastic.

Why is it just a comment?  Why aren't we using a list like this?

Reply:
Sorry, added Elastic. I don't know why things are like they are...